lib/omniauth/login_dot_gov/version.rb

```diff
@@ -1,5 +1,5 @@
 module OmniAuth
   module LoginDotGov
-    VERSION = '2.0.1'.freeze
+    VERSION = '2.0.2'.freeze
```

Comment:
I would vote for a minor version increase, since this is adding a new property/api field, but a patch version is ok too

```suggestion
    VERSION = '2.1.0'.freeze
```

Reply:
that makes sense , will update

Reply:
updated in https://github.com/18F/omniauth_login_dot_gov/pull/34/commits/032988d78904222f96b6b165e5902e64870bbeb4